epilogue_1_how_to_get_there_from_here.asciidoc

```diff
@@ -582,7 +582,8 @@ How should use cases interact across a larger system? Is it a problem for one to
 Is it a smell for a use case to use multiple repositories/aggregates, and why?::
     An aggregate is a consistency boundary, so if your use case needs to update
     two aggregates atomically (within the same transaction), then your
-    consistency boundary is wrong.
+   consistency boundary is wrong.  That might be fine ("practicality beats purity") but on the other hand
+   it may make your system harder to reason about.
```

Comment:
Am curious of @bobthemighty's opinion, but it feels to me that if you casually take this shortcut , it risks breaking assumptions and patterns that depend on each use case mutating only one aggregate.  So if you do it, then there should be a well-justified reason.

Reply:
I genuinely wouldn't modify multiple aggregates in a single transaction. This came up a lot at MADE, and my opinion was always the same: it's a hard no. I recognise that might mean rewriting a whole bunch of your domain, but your domain is wrong, and you should fix it.

It's tough to express that in a book because _obviously_ practicality trumps purity, but my unease about violating consistency boundaries is deep rooted.

Reply:
ok i've removed the practicality-beats-purity excuse.